plugins/woocommerce/includes/class-wc-install.php

```diff
@@ -1193,7 +1194,7 @@ public static function create_terms() {
 				'exclude-from-search',
 				'exclude-from-catalog',
 				'featured',
-				'outofstock',
+				ProductStockStatus::OUT_OF_STOCK,
```

Comment:
It's similar here: it feels like a mix of contexts.

Reply:
Also reverted in https://github.com/woocommerce/woocommerce/pull/54725/commits/c11c41ef0075c0af3fea750c1bb6ba7d37dfd33c